app/assets/stylesheets/views/signin.scss

```diff
@@ -209,6 +209,10 @@
     padding: var(--su-2);
     margin: var(--su-2);
 
+    & ul {
+      color: $black;
+    }
```

Comment:
This is working, in that the text shows up now against the white background in both light and dark themes. I'm going to tag @ludwiczakpawel in though to take a look, as I wonder if instead we want to theme the `error_explanation` UI, so that the background on this element isn't always white (it's currently hard-coded to `#f0f0f0`)

Reply:
Sorry for joining the party so late. 
I think this is good intermediate solution as we indeed don't style this component properly.

@forem/design has new Errors component in progress (see screenshots below) that should cover light and dark themes. I think it's matter of couple weeks till we have them ready to be used. For the time being, lets roll with this intermediate solution as it does fix the bug and fulfills the immediate need.

![image](https://user-images.githubusercontent.com/108287/151951502-92b18c78-def6-41ad-ad3c-9655eefc547b.png)

![image](https://user-images.githubusercontent.com/108287/151951521-ce440da5-004c-40bb-add1-72b8ac5f0124.png)